### ISSUE TITLE
Remove flail?

### ISSUE THREAD

**Issue by LordGolias** [LordGolias/Brytenwalda#24]
A normal weapon is on average 500 vertices on the most detailed view. Our flail has 6k vertices(!). Any army with this amount of vertices / weapon will heavily decrease the FPS of the battle.

AFAIK flail was *not* a weapon of this time, because 1. it had chains and 2. it required lots of metal.

Which bears the question: why are we using it?

**Comment by HooTmAnTW** [comment]
I think the flail is a nice idead BUT
1. I have stability problems to run warband or any mod with WSE, I played Imperia for some time so i konw how to set it up right. I am shure for me it is some Windows, application problem. Without WSE no crash at all.
2.Like Lord Golias sad it was not really a weapon at the time, in England.. and i love BW also for the history accuracy. Then about weapons you can not really say when, where, who.
3. FPS drops are bad. 
Actually the fail idea is great, if it would be possible with out WSE and maybe FPS problems. I would like to use them :) because of the fun.

**Comment by gdwitt** [comment]
The flail script only run in WSE mode which most players don't use; I haven't noticed any performance hits.
It has many merits:
1) It is the only blunt weapon that a cavalry unit can use to good effect. It it much longer than any hammer cold be. It allows a very skilled player to circle the enemy and pick them off one at a time.
2) It fits in with the subtheme of increasing usage of blunt weapons so as to attract players who like non-violent combat.
3) Attract more players
4) Chains (and chain mail )have been around since the Persian and Egyptian empires. They were widespread in ancient Rome. There is no doubt that some tried to use them for weapons.

**Comment by HooTmAnTW** [comment]
You are right, i tried it again, had no CTD with v 24 so far. I can understand you about the blunt weapons, i like this idea too. as an option in combat, the violent root of this time and this mod should be still great! I love it, because it is also very brutal sometimes! I saw a mod, adding decapitation with WSE, could be a lot of fun with a mix of also the standard BW Blood spread effect, for maybe flail! O.O

**Comment by HooTmAnTW** [comment]
ok i tired it for some time now, the idea is nice and makes fun. but the flail is kind of OP and i still have probs with WSE, exe. i like to play online and other rmods, so it is kind of bad to use and i have to switch the exe all the time..

**Comment by gdwitt** [comment]
WSE should not conflict with other games or other mods. It should launch from a desktop icon. I recommend that we discuss this issue in the WSE thread on the Brytenwalda forum. It needs to be clear to players what the pros and cons to it are.
Flails can be made weaker. I think they're a bit too fast for their type right now. Some players really go for OP. Most first person games feature OP players with OP weapons. You save the battle for your silly companions until they level up to 20. Meanwhile, you level up quicker as well.

**Comment by HooTmAnTW** [comment]
well thats ok but i am still for a WSE version and a Non WSE version. A friend of mine, does play this mod too. But he will never use WSE, he just dislikes this and i am shure there are more. I showed him the game with WSE (the flails) and he was like " I dont want to use WSE because of only this, it also looks not that good."
Well, i like the flails idea but they are not important for me, too. It is just more about the WSE thing, also the mod should work with out WSE just fine, so many more people would play, i think.

**Comment by HooTmAnTW** [comment]
I forgot to say, that i can not play online with the WSE exe, not on the neoGK servers like MountandSiege, so the NativeMod for MP won't work. I have to say i really want a non wse version for this mod now. sry
